test/unit_tests/augmentation/conftest.py

```diff
@@ -26,3 +33,24 @@ def random_batch(rng_seed, batch_size=5):
     rng = check_random_state(rng_seed)
     X = torch.from_numpy(rng.random((batch_size, 66, 51))).float().to(device)
     return X, torch.zeros(batch_size)
+
+
+class MockModule(torch.nn.Module):
```

Comment:
I'd remove it from `test_eeg_classifier` now that it has been moved here :)

Reply:
It is still used in `test_eeg_classifier` to test `predict` and `clone` functions. These functions are not tested in `test_base` elsewhere.

Reply:
Ahh, this `conftest.py` concerns only augmentation. Sorry, I though it was a general file for all tests.